.gitignore

```diff
@@ -216,4 +216,9 @@ docs/source/_auto_examples/*
 docs/source/_auto_examples
 examples/*_tmp
 *.loop3d
-m2l_data_tmp/*
\ No newline at end of file
+m2l_data_tmp/*
+docs/examples/m2l_data_tmp/*
+docs/source/sg_execution_times.rst
```

Comment:
we should delete this from the repository so that it is not tracked.

Reply:
https://github.com/Loop3D/map2loop/commit/36d59bce5bd7a8dfedd279c877a1d9b7ee6bf18b